Reneiski_Homework1/ReneiskiSelenium11/PageObjects/Pages/ButtonPage.cs

```diff
@@ -0,0 +1,21 @@
+﻿using OpenQA.Selenium;
+using ReneiskiSelenium11.Common.WebElements;
+
+namespace ReneiskiSelenium11.PageObjects.Pages
+{
+    public class ButtonPage : BasePage
+    {
+        private MyWebElement buttonPage = new MyWebElement(By.XPath("//*[contains(text(), 'Buttons')]"));
+        private MyWebElement dynamicClick = new MyWebElement(By.XPath("//*[(text()='Click Me')]"));
+        private MyWebElement doublClickButton = new MyWebElement(By.XPath("//*[@id='doubleClickBtn']"));
+        private MyWebElement rightClickButton = new MyWebElement(By.XPath("//*[@id='rightClickBtn']"));
+
+        public void DynamicClick() => dynamicClick.Click();
+
+        public void DoubleClick() => doublClickButton.ClickButton("doubleClickBtn");
+
+        public void RightClick() => rightClickButton.ClickButton("rightClickBtn");
```

Comment:
do you really need this ClickButton method? you may just use actions class from the driver inside a page. Currently, you kind of creating element twice: first time when you create  property doublClickButton, second time - inside ClickButton method

Reply:
Fixed